### ISSUE TITLE
Release 0.8.0 🏯 'ipfs' on PATH and bug fixing

### ISSUE THREAD

**Issue by hacdias** [ipfs-shipyard/ipfs-desktop#880]
After the huge release that 0.7.0 was (see #788), we need to start thinking about 0.8.0 😄 Here's the todo:

### What's left to be done

- [x] Add `ipfs` command line tools to the `PATH` (#727, https://github.com/ipfs-shipyard/ipfs-webui/pull/1016).
- [x] Figure out why we needed 'System Events' on macOS (#753).
- [x] Don't shutdown if there was an issue `fsck`ing the repository (#899, #835, #837).
- [x] Support macOS dark theme on native dialogs (#918)
- [x] Make dialogs more consistent and cohesive (#919) 
- [x] Avoid auto-update unnecessary errors (#892)
- [x] Update to Electron 5 (#917)
- [x] Allow settings when the daemon is offline (https://github.com/ipfs-shipyard/ipfs-webui/pull/1023)
- [x] Work with bandwidth metrics disabled (https://github.com/ipfs-shipyard/ipfs-webui/pull/1024)
- [x] Add folder via file-input fails in electron (#884)
- [x] Update Web UI hash
- [x] Update to `go-ipfs` v0.4.21.

### Postponed

- [ ] Hijack native macOS screenshot shortcut #879 
- [ ] Use a random port when the defaults are being used (#912)
- [ ] Investigate on how to make boot faster through [Electron Link](https://www.npmjs.com/package/electron-link)
- [ ] #785 preview PDFs

### After the release

- [x] Update Homebrew
- [x] Update Chocolatey

---

Please remember that this list list may change 🐉

**Comment by olizilla** [comment]
rather than a roadmap, lets update this issue to give it a pet name, and define just what is going in, like we did in https://github.com/ipfs-shipyard/ipfs-desktop/issues/788

**Comment by hacdias** [comment]
@olizilla could you check this issues, please:

- https://github.com/ipfs-shipyard/ipfs-webui/pull/1016
- https://github.com/ipfs-shipyard/ipfs-webui/pull/1023
- https://github.com/ipfs-shipyard/ipfs-webui/pull/1024

Also, how's the upload/1MB issue?

**Comment by olizilla** [comment]
Nice work! Very clear. The bad news is, there is a regression in go-ipfs 0.4.20 that means you can't add multiple individual files https://github.com/ipfs/go-ipfs/issues/6254 so either we wait, or roll back to 0.4.19... I'll see if there is an eta on another release for go-ipfs in the pipe.

Excting about electron 5 release, good work following up on the dep updates we need there.

**Comment by hacdias** [comment]
@olizilla in our code (webui), do we use the add command for more than one file at once? I thought we didn't. If we don't, that wouldn't be an issue, right?

Also, about waiting, I think: if it takes more than a week, we should rollback. Otherwise, we should wait.

**Comment by olizilla** [comment]
We use `ipfs.add` with an array of files dropped, or files selected in the file picker. Those use-cases fail if you drop or select more than 1 thing when using 0.4.20 now.

**Comment by olizilla** [comment]
see: https://github.com/ipfs/go-ipfs/issues/6254
also waiting on a release of ipfs-http-client with https://github.com/ipfs/js-ipfs-http-client/pull/980 in for the 1MB upload issue

**Comment by hacdias** [comment]
[v0.8.0](https://github.com/ipfs-shipyard/ipfs-desktop/releases/tag/v0.8.0) is out! 🎉
